main/templates/layouts/master_sidebar.html

```diff
@@ -19,13 +19,9 @@ <h1 class="title">{{ title }}</h1>
         <div class="col-12 mb-4">
           <div class="row justify-content-between align-items-center">
             <div class="col-12 col-md-8">
-              <div class="input-group rounded border-dark w-100" id="filterContainer">
-                <div class="form-floating" id="filter">
-                  <input type="search" id="id-search" class="form-control" placeholder="Filtra por {{ object_name }}">
-                  <label for="id-search"><i class="bi bi-search"></i> Filtra por {{ object_name }} ...</label>
-                </div>
-                <button class="btn btn-dark" id="filterBtn">Filtrar</button>
-              </div>
+              
+                {% include "components/search.html" with text="Buscar por usuario..." %}
```

Comment:
You shouldn't include this component like this because the text will be the same on each template that extends this layout. 
Maybe you can declare a variable and just pass the variable inside the include directive.

Reply:
You are right, I will change it

Reply:
I am trying to solve this but i get this in the view. I will continue solving this.

![image](https://user-images.githubusercontent.com/53529939/224518331-ec4b7cbd-e3cb-4890-a18d-fec371dbcf5d.png)

Reply:
I have fixed it finally. You can see it added to all list views with a context variable.